core/templates/core/irap_request/task_list.html

```diff
@@ -38,68 +38,81 @@
+                    <h2>2. Data</h2>
+                    <ul>
+                        <li>
+                            <a href="{% url 'core:request:using-data' object.pk %}">Use of data</a>{% task_status 'core:request:using-data' %}
+                        </li>
+                        <li>
+                            <a href="{% url 'core:request:identify-an-individual' object.pk %}">Personally identifiable information</a>{% task_status 'core:request:identify-an-individual' %}
+                        </li>
+                        <li>
+                            <a href="{% url 'core:request:number-of-records' object.pk %}">Processing of data</a>{% task_status 'core:request:number-of-records' %}
+                        </li>
+                        <li>
+                            <a href="{% url 'core:request:data-flow-diagram-needed' object.pk %}">Data flow diagram</a>{% task_status 'core:request:data-flow-diagram-needed' %}
+                        </li>
+                    </ul>
                 </li>
                 <li>
-                    <a href="{% url 'core:request:identify-an-individual' object.pk %}">Personally identifiable information</a>{% task_status 'core:request:identify-an-individual' %}
+                    <h2>3. Submit</h2>
+                    <ul>
+                        <li>
+                            <a href="{% url 'core:request:further-assurance' object.pk %}">Further assurance</a>{% task_status 'core:request:further-assurance' %}
+                        </li>
+                        <li>
+                            <a href="{% url 'core:request:check-and-submit' object.pk %}">Check and submit</a>
+                        </li>
+                    </ul>
                 </li>
-                <li>
-                    <a href="{% url 'core:request:number-of-records' object.pk %}">Processing of data</a>{% task_status 'core:request:number-of-records' %}
-                </li>
-                <li>
-                    <a href="{% url 'core:request:data-flow-diagram-needed' object.pk %}">Data flow diagram</a>{% task_status 'core:request:data-flow-diagram-needed' %}
-                </li>
-            </ul>
-        </li>
-        <li>
-            <h2>3. Submit</h2>
-            <ul>
-                <li>
-                    <a href="{% url 'core:request:further-assurance' object.pk %}">Further assurance</a>{% task_status 'core:request:further-assurance' %}
-                </li>
-                <li>
-                    <a href="{% url 'core:request:check-and-submit' object.pk %}">Check and submit</a>
-                </li>
-            </ul>
-        </li>
-    </ol>
+            </ol>
+        </div>
+        <div class="govuk-grid-column-one-third message-list">
+            <h2 class="govuk-heading-s">Notifications</h2>
+            {% if message_list|length == 0 %}
```

Comment:
```suggestion
            {% if message_list %}
```
switch if/else around